apps/marketplace/components/ListServices/utils.tsx

```diff
@@ -133,14 +134,15 @@ export const getServices = async (
 
   let servicesWithMetadata = await extractConfigDetailsForServices(results);
   if (chainId === 100 || chainId === 8453) {
```

Comment:
maybe could introduce a small helper that based on chainId returns true or false if marketplace is enabled? also can be used here `apps/marketplace/common-util/Details/index.tsx` and `apps/marketplace/pages/api/services.ts`